spyderlib/plugins/configdialog.py

```diff
@@ -661,13 +680,42 @@ def get_icon(self):
     def apply_settings(self, options):
         raise NotImplementedError
 
+    def prompt_restart_required(self):
+        """Prompt the user with a request to restart."""
+        restart_opt = self.restart_options
+        changed_opt = self.changed_options

```

Comment:
`restart_opt` -> `restart_opts` (because it's plural)
`changed_opt` -> `changed_opts`